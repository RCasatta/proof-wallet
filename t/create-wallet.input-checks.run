#!/bin/bash
set -e

$GLACIERSCRIPT create-wallet --regtest << INPUT 2>&1
y
y
y
y
y
y
1111111111 2222222222 3333333333 4444444444 5555555555 6666666666 1111111111 2222222222 3333333333 444444444
7777777777 7777777777 7777777777 7777777777 7777777777 7777777777 7777777777 7777777777 7777777777 7777777777
aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa
1111111111 2222222222 3333333333 4444444444 5555555555 6666666666 1111111111 2222222222 3333333333 4444444444
10c1 6550 67b0 10d3 61e2 1f60 8d09 e6ae 20d9 eb15 86fe 90b8 43b6 7f4b e102 e72
10c1 6550 67b0 10d3 61e2 1f60 8d09 e6ae 20d9 eb15 86fe 90b8 43b6 7f4b e102 e72b a
10c1 6550 67b0 10d3 61e2 1f60 8d09 e6ae 20d9 eb15 86fe 90b8 43b6 7f4b e102 e72q
10c1 6550 67b0 10d3 61e2 1f60 8d09 e6ae 20d9 eb15 86fe 90b8 43b6 7f4b e102 e72b
INPUT
