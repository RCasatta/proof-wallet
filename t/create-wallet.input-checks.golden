Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
Creating cold storage private key data.

Enter 100 dice rolls (example: 62543 16325 21341...) Spaces are OK, and will be ignored:
Error: You must provide at least 100 dice rolls.
Enter 100 dice rolls (example: 62543 16325 21341...) Spaces are OK, and will be ignored:
Error: Dice rolls must be between 1 and 6.
Enter 100 dice rolls (example: 62543 16325 21341...) Spaces are OK, and will be ignored:
Error: Dice rolls must be numbers between 1 and 6.
Enter 100 dice rolls (example: 62543 16325 21341...) Spaces are OK, and will be ignored:
Enter at least 64 characters of computer entropy. Spaces are OK, and will be ignored:
Error: Computer entropy must be at least 64 characters long.
Enter at least 64 characters of computer entropy. Spaces are OK, and will be ignored:
Error: Computer entropy must contain an even number of characters.
Enter at least 64 characters of computer entropy. Spaces are OK, and will be ignored:
Error: Illegal character. Computer entropy must be composed of hexadecimal characters only (0-9, a-f).
Enter at least 64 characters of computer entropy. Spaces are OK, and will be ignored:

BIP39 Mnemonic Phrase: 
1. mouse
2. scout
3. chest
4. give
5. trim
6. veteran
7. keen
8. arrive
9. test
10. advice
11. warrior
12. reduce
13. silent
14. useful
15. oak
16. embrace
17. drift
18. curve
19. large
20. poet
21. embody
22. theme
23. focus
24. wolf

xpub:
tpubD6NzVbkrYhZ4YTjZqdGdMpqtd7BFKkERJcK3hjQx4gtHC3xowC5EKAsHY5tgx1coSC6sbw6X7Ngc1AFrVEzgwFFAF8pbsU5n28yYiZrxTAg

QR code for xpub written to xpub.png
